openfeature/event_executor.go

```diff
@@ -42,42 +57,50 @@ func newEventExecutor(logger logr.Logger) *eventExecutor {
 	return &executor
 }
 
-// scopedCallback is a helper struct to hold client name associated callbacks.
+// ScopedCallback is a helper struct to hold client name associated callbacks.
 // Here, the scope correlates to the client and provider name
-type scopedCallback struct {
+type ScopedCallback struct {
 	scope     string
 	callbacks map[EventType][]EventCallback
 }
 
-type eventPayload struct {
-	event   Event
-	handler FeatureProvider
+func (s *ScopedCallback) Callbacks() map[EventType][]EventCallback {
+	return s.callbacks
+}
+
+func (s *ScopedCallback) Scope() string {
+	return s.scope
 }
 
-func newScopedCallback(client string) scopedCallback {
-	return scopedCallback{
+func newScopedCallback(client string) ScopedCallback {
+	return ScopedCallback{
 		scope:     client,
 		callbacks: map[EventType][]EventCallback{},
 	}
 }
 
+type eventPayload struct {
+	event   Event
+	handler FeatureProvider
+}
+
 // providerReference is a helper struct to store FeatureProvider with EventHandler capability along with their
-// shutdown semaphore
+// Shutdown semaphore
```

Comment:
I see `shutdown` replaced with `Shutdown` in many comments (here and below) - I'm assuming a find/replace error?

Reply:
yeah thank you